### PR TITLE
[NFC] Fix issue-61440 test formatting

### PR DESCRIPTION
**Pull request by jreference**
Added a test as part of fixing #61440 but the format was slightly irregular. Fixed the formatting of the test. @slavapestov Sorry, was not able to fix the test in the prior PR

Related: https://github.com/apple/swift/pull/65434

### CODE REVIEW

**Comment by LucianoPAlmeida** [comment]
@swift-ci Please smoke test

File: test/Sema/generic-subscript.swift [hunk 1]
```diff
@@ -1,44 +1,34 @@
...
-    } 
+protocol P {
+  subscript<Value>(x: Value) -> Int  // expected-note {{protocol requires subscript with type '<Value> (Value) -> Int'; do you want to add a stub?}}
+  {
+    get
+  }
 }
 
-struct S:P // expected-error {{type 'S' does not conform to protocol 'P'}}
+struct S: P  // expected-error {{type 'S' does not conform to protocol 'P'}}
```
Comment (AnthonyLatsis) [inline test/Sema/generic-subscript.swift:10]: Not that it really matters but one space before a trailing comment in enough.

**Review by AnthonyLatsis — COMMENTED** [review]
Thanks for taking on the issue! Line breaks before `{` are still inconsistent though; could you fix that?

Please consider using more idiomatic and descriptive commit messages. For example, `Fix test formatting after #<pull request id>`.

**Review by slavapestov — APPROVED** [review]

File: test/Sema/generic-subscript.swift (hunk 1 above)
Comment (jreference) [reply test/Sema/generic-subscript.swift:10]: > Thanks for taking on the issue! Line breaks before `{` are still inconsistent though; could you fix that?
> 
> Please consider using more idiomatic and descriptive commit messages. For example, `Fix test formatting after #<pull request id>`.

Got it, will do in my next PR!

**Pull request merged**